src/databricks/labs/remorph/resources/reconcile/dashboards/reconciliation_metrics/11_0_recon_details_pivot.sql

```diff
@@ -0,0 +1,34 @@
+/* --height 6 --width 6 */
+WITH tmp AS (
+    SELECT
+        recon_table_id,
+        inserted_ts,
+        recon_type,
+        explode(data) AS data,
+        row_number() OVER (PARTITION BY recon_table_id, recon_type ORDER BY recon_table_id) AS rn
+    FROM
+        remorph.reconcile.details
+    WHERE
+        recon_type != 'schema'
+)
+SELECT
+    main.recon_id AS dd_recon_id,
+    main.source_table.`catalog` AS source_catalog,
+    main.source_table.`schema` AS source_schema,
+    main.source_table.table_name AS source_table_name,
+    CASE
+        WHEN COALESCE(main.source_table.catalog, '') <> '' THEN CONCAT(main.source_table.catalog, '.', main.source_table.schema, '.', main.source_table.table_name)
```

Comment:
This case statement can be converted as below, avoiding the Unnecessary use of `Coalesece`
```
 IF(
                     ISNULL(source_catalog),
                     CONCAT_WS('.', source_schema, source_table_name),
                     CONCAT_WS(
                             '.',
                             source_catalog,
                             source_schema,
                             source_table_name
                     )
             ) AS source_table,
```